src/components/Trends/Explore/TrendsExploreAdditionalInfo.js

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from 'react'
+import { compose } from 'recompose'
+import { graphql } from 'react-apollo'
+import { Tabs } from '@santiment-network/ui'
+import {
+  DAY,
+  getTimeIntervalFromToday,
+  ONE_DAY_IN_MS
+} from '../../../utils/dates'
+import { ALL_INSIGHTS_BY_TAG_QUERY } from '../../../queries/InsightsGQL'
+import { NEWS_QUERY } from '../../News/NewsGQL'
+import { getInsightTrendTagByDate } from '../../Insight/InsightsTrends'
+import InsightCard from '../../Insight/InsightCard'
+import News from '../../News/News'
+import styles from './TrendsExploreAdditionalInfo.module.scss'
+
+const NEWS_INDEX = 'News'
+const INSIGHTS_INDEX = 'Insights'
+
+const TrendsExploreAdditionalInfo = ({ news, insights }) => {
+  let [selectedTab, setSelectedTab] = useState(null)
+
+  function handleSelectTab (tab) {
+    if (tab !== (selectedTab || defaultSelectedTab)) {
+      setSelectedTab(tab)
+    }
+  }
+
+  const newsTab = {
+    index: NEWS_INDEX,
+    content: `${NEWS_INDEX} (${news.length})`
+  }
+
+  const insightsTab = {
+    index: INSIGHTS_INDEX,
+    content: `${INSIGHTS_INDEX} (${insights.length})`
+  }
+
+  const tabs = []
+  if (insights.length > 0) tabs.push(insightsTab)
+  if (news.length > 0) tabs.push(newsTab)
+
+  if (tabs.length === 0) return null
+  const defaultSelectedTab = tabs[0].index
+
+  return (
+    <section className={styles.wrapper}>
+      <Tabs
+        options={tabs}
+        defaultSelectedIndex={defaultSelectedTab}
+        onSelect={handleSelectTab}
+      />
+      <div className={styles.tabsContent}>
+        {(selectedTab || defaultSelectedTab) === NEWS_INDEX && (
+          <News data={news} />
+        )}
+        {(selectedTab || defaultSelectedTab) === INSIGHTS_INDEX && (
+          <div className={styles.insights}>
+            {insights.map(insight => (
+              <InsightCard
+                small
+                grey
+                withAuthorPic={false}
+                {...insight}
+                key={insight.id}
+                className={styles.insight}
+              />
+            ))}
+          </div>
+        )}
+      </div>
+    </section>
+  )
+}
+
+const getTrendsTags = numberOfLastDays =>
+  Array.from({ length: numberOfLastDays }, (_, idx) =>
+    getInsightTrendTagByDate(new Date(Date.now() - ONE_DAY_IN_MS * idx))
+  )
+
+const filterInsights = (insights = [], word) =>
+  insights.filter(({ tags }) => tags.find(({ name }) => name === word))
```

Comment:
`.find` returns found item if it exists. In our case, we just need to get a boolean value -> `.some`